app/src/main/res/layout/fragment_character.xml

```diff
@@ -9,26 +9,30 @@
 
     <ImageView
         android:id="@+id/imageView"
-        android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_width="wrap_content"
         android:minWidth="@dimen/super_hero_image_width_frag"
-        android:minHeight="@dimen/super_hero_image_height_frag"
         android:layout_gravity="center"
         app:srcCompat="@android:color/holo_blue_bright" />
 
     <TextView
         android:id="@+id/text_characterName"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="@dimen/activity_horizontal_margin"
+        android:layout_marginRight="@dimen/activity_horizontal_margin"
         android:layout_gravity="center"
-        android:textSize="@dimen/title_text_size_frag"
-        android:text="Your name" />
+        android:textAlignment="center"
+        android:textSize="@dimen/title_text_size_frag" />
 
     <TextView
         android:id="@+id/text_characterDesc"
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:textSize="@dimen/body_text_size_frag"
-        android:text="Your description" />
+        android:layout_marginLeft="@dimen/activity_horizontal_margin"
+        android:layout_marginRight="@dimen/activity_horizontal_margin"
+        android:layout_gravity="center"
+        android:textAlignment="center"
+        android:textSize="@dimen/body_text_size_frag" />
 
 </LinearLayout>
```

Comment:
to avoid this annoying git error, configure android studio like this
https://stackoverflow.com/questions/24093609/how-can-i-configure-android-studio-to-add-new-line-at-end-of-file

Reply:
Done